src/TodoList.js

```diff
@@ -1,5 +1,28 @@
 class TodoList {
+  constructor() {
+    this.todoList = [];
+  }
 
+  displayTodoList() {
+    return this.todoList;
+  }
+
+  create(text) {
+    const todo = {
+      id: this.todoList.length + 1,
+      text: text,
+      status: "incomplete",
+    };
+
+    this.todoList.push(todo);
+  }
+
+  isComplete(status) {
```

Comment:
The name of this method suggests it will return a boolean, `isComplete() -> yes or no / true or false`

I think you probably want a clearer name - this method is returning the todos that have a certain status that gets passed in as a parameter, so maybe `getTodosWithStatus(status)`